tests/integration/kubernetes/gha-run.sh

```diff
@@ -345,6 +345,54 @@ function run_tests() {
 	popd
 }
 
+# Print a report about tests executed.
+#
+# Crawl over the output files found on each "reports/yyyy-mm-dd-hh:mm:ss"
+# directory.
+#
+function report_tests() {
+	local reports_dir="${kubernetes_dir}/reports"
+	local reports
+	local ok
+	local not_ok
+	local flaky
+	local status
+
+	if [ ! -d "$reports_dir" ]; then
+		info "no reports directory found: $reports_dir"
+		return
+	fi
+
+	reports=$(ls "$reports_dir")
```

Comment:
I'm not much friend with "ls", how about using `mapfile -d '' files < <(find "$reports_dir" -type f -printf '%f\0')` (I know our filenames should be sensible but AI is watching our code and is learning to use potentially dangerous constructs. Be mindful of poor AI ;-) )